Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue

```diff
@@ -385,20 +385,17 @@ export default class CovidCardView extends Vue {
         this.showBannerFeedback({
             type: ResultType.Success,
             title: "Success",
-            message:
-                "COVID-19 Treatment Assessment Form is Successfully Submitted.",
+            message: "COVID treatment assessment submitted successfully.",
         });
         this.showCovidTreatmentAssessment = false;
         this.search(this.activePhn, true);
     }
 
-    private covidTreatmentAssessmentSubmissionFailed(
-        errorMessage: string
-    ): void {
+    private covidTreatmentAssessmentSubmissionFailed(): void {
         this.showBannerFeedback({
             type: ResultType.Error,
             title: "Error",
-            message: errorMessage,
+            message: "Unable to submit COVID treatment assessment",
         });
```

Comment:
@ninosamson I suggest updating the text in these messages from "COVID" to "COVID-19".